modules/gapi/include/opencv2/gapi/render/render.hpp

```diff
@@ -28,17 +78,21 @@ namespace draw
 {
 
 /**
- * A structure allows using freetype library for text rendering
+ * @brief This structure specifies which FreeType font to use by FText primitives.
  */
 struct freetype_font
 {
     /*@{*/
-    std::string path; //!< The path to font file (.ttf)
+    std::string path; //!< The path to the font file (.ttf)
```

Comment:
Could you remove `(.ttf)` It's my mistake. Render works with all font formats. Not only `TrueType`

Reply:
Can you address it on your own in the following `#include` review?